### PR TITLE
Update frequency-plans.md

### PR DESCRIPTION
**Pull request by nestorayuso**
add note about US915 devices personalization

### CODE REVIEW

File: _content/lorawan/frequency-plans.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ Downlink:
 7. **926.9** - SF7BW500 to SF12BW500
 8. **927.5** - SF7BW500 to SF12BW500
 
+Note that The Things Network uses 2nd Sub-Band only (channels 8,9,10,11,12,13,14,15,65). If your devices use OTAA and ADR, this will be configured automatically when they join (expect some join retries and the first ~8 packets to fail). If your devices use ABP, you'll need to program the specific channels into the devices in order to make them work with TTN.
```
Comment (johanstokking) [inline _content/lorawan/frequency-plans.md]: - Use `>` indentation
- FSB 2, and only on the public community network (2x)
- Channels `[8..15, 65]` or `8 to 15 and 65`
- Why would the first 8 packets fail?

**Review by johanstokking — CHANGES_REQUESTED** [review]
Thanks! Few comments.

File: _content/lorawan/frequency-plans.md (hunk 1 above)
Comment (nestorayuso) [reply _content/lorawan/frequency-plans.md]: An unpersonalized device will try to join using the whoole 64+8 channels, so expect ~8 join retials before matching one join in the 2nd sub-band.

To personalize the device you need two MAC commands (LinkADRReq) one goes in the join accept and the other goes in the RX window of first received uplink, so expect ~8 first uplinks to fail before matching one uplink in the 2nd sub-band.

I also obseved that if the device has ADR disabled, the personalization does not happen.

Comment (johanstokking) [reply _content/lorawan/frequency-plans.md]: >An unpersonalized device will try to join using the whoole 64+8 channels, so expect ~8 join retials before matching one join in the 2nd sub-band.

That depends on the channel hopping strategy on join. Best practices are to try subband after subband with a random channel, not trying all channels in the first subband first.

Comment (nestorayuso) [reply _content/lorawan/frequency-plans.md]: yes, that's why you can expect up to 8 joins to get joined, you need not wait up to 56 joins

Comment (johanstokking) [reply _content/lorawan/frequency-plans.md]: According to my beer mat calculation; with all join requests resulting in a join accept, the expected number of retries is 1 (FSB 1 attempt fails).

File: _content/lorawan/frequency-plans.md [hunk 2]
```diff
@@ -61,6 +61,8 @@ Downlink:
 7. **926.9** - SF7BW500 to SF12BW500
 8. **927.5** - SF7BW500 to SF12BW500
 
+>Note that The Things Network uses 2nd Sub-Band only (channels 8 to 15 and 65). If your devices use OTAA and ADR, this will be configured automatically when they join (expect some join retries and the first ~8 packets to fail). If your devices use ABP, you'll need to program the specific channels into the devices in order to make them work with TTN.
```
Comment (johanstokking) [inline _content/lorawan/frequency-plans.md:64]: We do send MAC commands to disable the first 8 channels

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Pull request closed without merging**